Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [feature] Added `FIRQuery.limitedToLast(_:limit)`, which returns the last
```

Comment:
The swift name for this is `limit(toLast:)` and the Objective-C is `queryLimitedToLast:`. We seem to be preferring the Swift names, so looking below it seems the most consistent way to spell this would be:

> Added a `limit(toLast:)` query operator, which returns the last matching documents up to the given limit.

?

Reply:
Done.